allennlp/commands/evaluate.py

```diff
@@ -37,7 +37,11 @@ def add_subparser(self, parser: argparse._SubParsersAction) -> argparse.Argument
             "input_file", type=str, help="path to the file containing the evaluation data"
         )
 
-        subparser.add_argument("--output-file", type=str, help="path to output file")
+        subparser.add_argument("--output-file", type=str, help="path to write the metrics to")
+
+        subparser.add_argument(
+            "--predictions-output-file", type=str, help="path to write the predictions to"
```

Comment:
```suggestion
            "--predictions-output-file", type=str, help="path to write the predictions to (if desired)"
```

Or something like that.  Or maybe just change the `description` above to add something like "and optionally save the metrics and/or predictions to disk".